conf/messages.cy

```diff
@@ -343,9 +382,12 @@ form.upload-multiple-files.continue=Yn eich blaen
 error.file-upload.required=Dewiswch ffeil
 error.file-upload.invalid-size-small=Mae’r ffeil dan sylw yn wag
 error.file-upload.invalid-size-large=Mae’n rhaid i’r ffeil dan sylw fod yn llai na {0}MB
-error.file-upload.invalid-type=Mae’n rhaid i’r ffeil dan sylw fod ar ffurf .jpg, .png, .tiff, .pdf, .txt, .msg, Open Document Format (ODF) neu Microsoft Office
+error.file-upload.invalid-type=Mae’n rhaid i’r ffeil dan sylw fod ar ffurf .jpg, .png, .tiff, .pdf, .txt, .msg, Fformat Dogfen Agored (ODF), neu Microsoft Office
 error.file-upload.quarantine=Mae feirws yn y ffeil dan sylw – uwchlwythwch un wahanol
 error.file-upload.unknown=Nid oedd modd uwchlwytho’r ffeil dan sylw
+error.file-upload.remove.try-again=Nid oedd modd tynnu’r ffeil dan sylw – rhowch gynnig arall arni
+error.file-upload.duplicate=Mae’r ffeil dan sylw eisoes wedi’i uwchlwytho
+error.file-upload.try-again=rhowch gynnig arall arni
```

Comment:
Still needs to have the previous part of the error so full message will be: 
error.file-upload.try-again= Nid oedd modd uwchlwytho’r ffeil dan sylw  - rhowch gynnig arall arn

Reply:
It wouldn't let me comment beside the actual lines but these also need the translation added: 

view.upload-file.help.fileFormat4=Microsoft (Word, Excel or PowerPoint)
view.upload-file.help.fileFormat5=Open Document Format (ODF)

We can use the content from the Beta start page:

**Microsoft (Word, Excel neu PowerPoint)**
**Fformat Dogfen Agored (ODF)**